src/js/control-bar/text-track-controls/subs-caps-menu-item.js

```diff
@@ -14,20 +15,33 @@ import {assign} from '../../utils/obj';
 class SubsCapsMenuItem extends TextTrackMenuItem {
 
   createEl(type, props, attrs) {
-    let innerHTML = `<span class="vjs-menu-item-text">${this.localize(this.options_.label)}`;
+    const el = super.createEl(type, props, attrs);
 
-    if (this.options_.track.kind === 'captions') {
-      innerHTML += `
-        <span aria-hidden="true" class="vjs-icon-placeholder"></span>
-        <span class="vjs-control-text"> ${this.localize('Captions')}</span>
-      `;
+    while (el.firstChild) {
+      el.removeChild(el.firstChild);
     }
 
-    innerHTML += '</span>';
+    const parentSpan = createEl('span', {
+      className: 'vjs-menu-item-text'
+    });
+
+    parentSpan.appendChild(document.createTextNode(this.localize(this.options_.label)));
```

Comment:
Seems like there we can do the same as for the audio track menu item. Instead of removing all children, we grab the span that's created and append to it, since it seems like it gets created as we want.